mlton/main/main.fun

```diff
@@ -383,11 +383,18 @@ fun makeOptions {usage} =
             reportAnnotation (s, flag,
                               Control.Elaborate.processEnabled (s, false))))
        end,
-       (Expert, "drop-pass", " <pass>", "omit optimization pass",
+       (Expert, "disable-pass", " <pass>", "disable optimization pass",
         SpaceString
         (fn s => (case Regexp.fromString s of
                      SOME (re,_) => let val re = Regexp.compileDFA re
-                                    in List.push (dropPasses, re)
+                                    in List.push (doPasses, (re, false))
+                                    end
+                   | NONE => usage (concat ["invalid -drop-pass flag: ", s])))),
+       (Expert, "enable-pass", " <pass>", "enable optimization pass",
```

Comment:
The command-line options should be specified in alphabetical order, so `enable-pass` should go after `enable-ann`.